RecoLocalCalo/EcalRecAlgos/BuildFile.xml

```diff
@@ -14,7 +14,7 @@
 
 <use   name="eigen"/>
 
-<Flags CXXFLAGS="-Ofast"/>
+<flags CXXFLAGS="-O3"/>
```

Comment:
IIRC -O3 is the default. So, it should not be used explicitly in the build file.

@VinInn may be able to comment why Ofast is slower. Maybe the comparison changes depending on the architecture.